saleor/graphql/account/types.py

```diff
@@ -153,6 +154,37 @@ def resolve_order_line(root: models.CustomerEvent, info):
         return None
 
 
+class ServiceAccount(MetadataObjectType, CountableDjangoObjectType):
+    permissions = graphene.List(
+        PermissionDisplay, description="List of service's permissions."
+    )
+    created = graphene.DateTime(description="Created date")
+    is_active = graphene.Boolean()
+    name = graphene.String()
+
+    class Meta:
+        description = "Represents service account data."
+        interfaces = [relay.Node]
+        model = models.ServiceAccount
+        permissions = ("account.manage_service_account",)
+        only_fields = ["name" "permissions", "created", "is_active", "auth_token", "id"]
+
+    @staticmethod
+    def resolve_permissions(root: models.ServiceAccount, _info, **_kwargs):
+        permissions = root.permissions.prefetch_related("content_type").order_by(
+            "codename"
+        )
+        return format_permissions_for_display(permissions)
+
+    @staticmethod
+    def resolve_auth_token(root: models.ServiceAccount, _info, **_kwargs):
+        return "*" * 6 + root.auth_token[-4:]
```

Comment:
Do we want to expose the length of it?

Reply:
I don't think so. This is the reason why I show only 10 characters. 
@maarcingebala What do you think?

Reply:
I don't think we should be adding the `*` server-side. UI may prefer to use other characters like (`●` is a popular choice).